README.md

```diff
@@ -198,6 +198,40 @@ One benefit of using parameter resolvers instead of hard coding values like VPC
 IDs and resource ARNs is that the same configuration works cross
 region/account, even though the resolved values will be different.
 
+### Cross-account parameter resolving
+
+One way to resolve parameter values from different accounts to the one StackMaster runs in, is to
+assume a role in another account with the relevant IAM permissions to execute successfully.
+
+This is supported in StackMaster by specifying the `role` and `account` properties for any
+parameter resolver in the stack's parameters file.
+
+```yaml
+vpc_peering_id:
+  role: cross-account-parameter-resolver
+  account: 1234567890
+  stack_output: vpc-peering-stack-in-other-account/peering_name
```

Comment:
It looks like this'd work with other parameter resolvers. If so should we add some examples in? SSM parameter store for instance:

```yaml
my_secret:
  role: cross-account-parameter-resolver
  account: 1234567890
  parameter_store: ssm_parameter_name
```

Reply:
Good idea.

I also tweaked the wording a little to be explicit about which parameter resolvers support assuming a role in 31bbaa3. I.e., all of them!